maro/cli/grass/image.py

```diff
@@ -6,19 +6,24 @@
 from maro.cli.utils.checkers import check_details_validity
 from maro.cli.utils.details import load_cluster_details
 from maro.cli.utils.lock import lock
+from maro.utils.exception.cli_exception import BadRequestError
 
 
-@check_details_validity(mode='grass')
+@check_details_validity
 @lock
-def push_image(cluster_name: str, image_name: str, image_path: str, remote_context_path: str, remote_image_name: str,
-               **kwargs):
+def push_image(
+    cluster_name: str, image_name: str, image_path: str, remote_context_path: str, remote_image_name: str,
+    **kwargs
+):
     cluster_details = load_cluster_details(cluster_name=cluster_name)
```

Comment:
It seems you don't need call this line:
 cluster_details = load_cluster_details(cluster_name=cluster_name)

Reply:
actually, we need it because we will need to check the 'mode' here, and the 'mode' is stored in the cluster_details